src/api/app/views/webui/package/_buildstatus.html.erb

```diff
@@ -1,9 +1,7 @@
-<% @package.buildresults.each_pair do |package, results| %>
-  <h3><%= package %></h3>
-  <div id="package_buildstatus">
-    <% if !@project.repositories.any? %>
-      <%= render "no_repositories" %>
-    <% else %>
+<% if @project.repositories.any? %>
```

Comment:
Would this work instead? Since we were talking about removing the logic from the views at some point, maybe this is something we can do now? Otherwise there is this logic stored in the view: no project.repositories => no build results. Best would be that `@package.buildresults` returns an empty hash if there are no build results.


```ruby
@package.buildresults.any?
```